Doc/library/pprint.rst

```diff
@@ -85,6 +86,12 @@ Functions
       integers will be formatted with the ``_`` character for a thousands separator,
       otherwise underscores are not displayed (the default).
 
+   :param bool block_style:
+      If ``True``,
+      opening parentheses and brackets will be followed by a newline and the
+      following content will be indented by one level, similar to block style
+      JSON formatting. This option is not compatible with *compact*.
```

Comment:
> This option is not compatible with *compact*.

A passing thought: Perhaps introduce a *mode* (str) argument, and deprecate *compact* in favour of it? It's not great to have two conflicting Booleans, though it may be the best option for now.